src/main/java/seedu/duke/parser/CommandParser.java

```diff
@@ -113,6 +118,7 @@ private static Command getCommand(String commandWordInput, String parameterInput
             command = new ByeCommand();
             break;
         default:
+            parserLogger.log(Level.WARNING, "An invalid command error is caught in this command");
```

Comment:
Looks good! The log is tagged to the right level as it is an exception, so it should be a log warning.